client/themes/morning.css

```diff
@@ -46,8 +46,22 @@ body {
 	padding: 15px 16px;
 }
 
+/* stylelint-disable */
 #chat .search::-webkit-input-placeholder {
-	color: #99a2b4;
+/* stylelint-enable */
+	color: #99a2b4 !important;
+	opacity: .5;
+}
+
+/* stylelint-disable */
+#chat .search::-moz-placeholder {
+/* stylelint-enable */
```

Comment:
At some point, theme management should be nice enough that one can add a theme easily. When this happens, themes should not be linted on our end (except the default theme(s) it ships with) but dealt with at will by their authors.
In the meantime, this seems fine.